src/Directory.Build.props

```diff
@@ -1,7 +1,7 @@
 <Project>
 
   <PropertyGroup>
-    <VersionPrefix>7.3.0-preview</VersionPrefix>
+    <MinVerDefaultPreReleaseIdentifiers>preview</MinVerDefaultPreReleaseIdentifiers>
```

Comment:
https://github.com/adamralph/minver#can-i-change-the-default-pre-release-identifiers-from-alpha0-to-something-else